src/modules/user/dto/requests/update-profile.dto.ts

```diff
@@ -0,0 +1,14 @@
+import { IsEmail, IsNotEmpty, IsString } from 'class-validator';
+
+export class UpdateProfileDto {
+  @IsEmail()
+  email?: string;
+
```

Comment:
You must mark this as optional with @IsOptional() decorator

Reply:
Done